components/auth/LogIn.js

```diff
@@ -87,12 +88,12 @@ const LogIn = () => {
             value={inputs.password}
             onChange={handleChange}
           />
-        </label>
-        <button disabled={loading} type="submit">
-          Sign the heck in
-        </button>
-      </fieldset>
-    </form>
+        </StyledLabel>
+        <Button styleProp="primary" disabled={loading} type="submit">
```

Comment:
Personally I think this is not a good name for a prop. It doesn't say anything. A better name would be styleButton, or btnType, something in that sort

Reply:
I will call it variant.